deploy/mimoto/README.md

```
@@ -6,6 +6,8 @@ Helm chart for installing MOSIP mimoto Application.
 
 ```console
 $ helm repo add mosip https://mosip.github.io
```

markdownlint-cli2 (0.20.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)